opm/autodiff/CPRPreconditioner.hpp

```diff
@@ -256,6 +267,44 @@ createEllipticPreconditionerPointer(const M& Ae, double relax,
 #endif
 } // end namespace
 
+    struct CPRParameter
+    {
+        double cpr_relax_;
+        double cpr_solver_tol_;
+        int cpr_ilu_n_;
+        int cpr_max_ell_iter_;
+        bool cpr_use_amg_;
+        bool cpr_use_bicgstab_;
+        bool cpr_solver_verbose_;
+
+        CPRParameter() { reset(); }
+
+        CPRParameter( const parameter::ParameterGroup& param)
```

Comment:
Did you forget to make this constructor explicit, or do you intend automatic conversion?